### PR TITLE
Ruby - splat range into an array

### PR DESCRIPTION
**Pull request by mikeastock**

### CODE REVIEW

File: ruby/splat-range-into-array.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Splatting a range into an Arrray
+
+Ruby allows you to splat a range into an array. This can be useful when you need
+to expand a range into an array. For example:
+
+```ruby
+[*("a".."z")]
+#=> ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+
+[*(1..10)]
```
Comment (JuanitoFatas) [inline ruby/splat-range-into-array.md]: Just a note: `[*1..10]` also works.

File: ruby/splat-range-into-array.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Splatting a range into an Arrray
+
+Ruby allows you to splat a range into an array. This can be useful when you need
+to expand a range into an array. For example:
+
+```ruby
+[*("a".."z")]
+#=> ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
```
Comment (muescha) [inline ruby/splat-range-into-array.md]: better add newline - so that not need to scroll vertically

**Comment by zamith** [comment]
Not sure if it is interesting, but there are more ways of achieving the same goal. Here's an [answer](http://stackoverflow.com/questions/191329/correct-way-to-populate-an-array-with-a-range-in-ruby/6587096#6587096) I wrote a while ago in stack overflow.

**Comment by muescha** [comment]
@zamith i think your answers from SO should be added

File: ruby/splat-range-into-array.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Splatting a range into an Arrray
+
+Ruby allows you to splat a range into an array. This can be useful when you need
+to expand a range into an array. For example:
+
+```ruby
+[*("a".."z")]
```
Comment (iwz) [inline ruby/splat-range-into-array.md:7]: no need to go `a..z`. could do `a..f` and get the same point across

**Pull request closed without merging**